### PR TITLE
feat: Create a standalone ClusterLogging app within operate-first project

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/apps/issues/174

Depends on: https://github.com/operate-first/apps/pull/173
Merge together with: https://github.com/operate-first/continuous-deployment/pull/97

### CODE REVIEW

**Review by HumairAK — APPROVED** [review]

**Review by 4n4nd — APPROVED** [review]

File: overlays/quicklab/operate-first/cluster-logging.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: cluster-logging
+spec:
+  source:
+    path: cluster-logging/overlays/quicklab/
```
Comment (HumairAK) [inline overlays/quicklab/operate-first/cluster-logging.yaml:8]: we may need to add the crds/subs/etc. in the `operate-first/apps/cluster-resources/` location relevant to this CR into the quicklab environments

Comment (HumairAK) [reply overlays/quicklab/operate-first/cluster-logging.yaml:8]: same with crc

Comment (tumido) [reply overlays/quicklab/operate-first/cluster-logging.yaml:8]: yeah...

Comment (tumido) [reply overlays/quicklab/operate-first/cluster-logging.yaml:8]: I updated it - a different route though as per your suggestion - autosync only in MOC, app left as is for other downstreams

**Review by HumairAK — APPROVED** [review]

**Pull request merged**